package.json

```diff
@@ -41,6 +41,7 @@
     "ethers": "^5.6.9",
     "idriss-crypto": "^1.9.1",
     "lit": "^3.1.2",
+    "lodash.debounce": "^4.0.8",
     "lodash.isequal": "^4.5.0",
```

Comment:
I am wondering if we need this granularity in lodash modules, maybe we can get whole lodash package and just import not via barell file so we don't get unncessary bundle.

`import debounce from 'lodash/debounce';`

Reply:
sure, makes sense 👌